src/Jackett.Common/Indexers/BakaBT.cs

```diff
@@ -46,6 +48,21 @@ public BakaBT(IIndexerConfigurationService configService, Utils.Clients.WebClien
             Encoding = Encoding.UTF8;
             Language = "en-us";
             Type = "private";
+
+            InitCategoryMappings();
+        }
+
+        private void InitCategoryMappings()
+        {
+            AddCategoryMapping(1, TorznabCatType.TVAnime, "Anime Series");
```

Comment:
Move this code at the end of `ConfigurationDataBasicLogin` and remove the `InitCategoryMappings` method.

Reply:
will get around to it on weekend

Reply:
Done